### PR TITLE
OPS-665: Fix neo4j crash OutOfMemoryError

### PR DESCRIPTION
**Pull request by thibveni**

### CODE REVIEW

File: charts/neo4j-internal/values.yaml [hunk 1]
```diff
@@ -94,3 +94,16 @@ ingressRoute:
...
       # - chart-example.local
+
+  # Jvm configuration for Neo4j
+  jvm:
+    # If true any additional arguments are added after the Neo4j default jvm arguments.
+    # If false Neo4j default jvm arguments are not used.
+    useNeo4jDefaultJvmArguments: true
+    # additionalJvmArguments is a list of strings. Each jvm argument should be a separate element:
+    additionalJvmArguments:
+    - "-XX:+ExitOnOutOfMemoryError"
```
Comment (enys) [inline charts/neo4j-internal/values.yaml:105]: That's indeed what I had in mind when I mentioned jvm arts. Have you tested ?

Comment (thibveni) [reply charts/neo4j-internal/values.yaml:105]: nope, should i
- allocate a low memory in values
- and run it in devops-dev and see it's triggered ?

Comment (enys) [reply charts/neo4j-internal/values.yaml:105]: You can use a preview env to test, at least that neo starts

Comment (thibveni) [reply charts/neo4j-internal/values.yaml]: neo4j@neo4j-pr-42-0:~$ for proc in /proc/*/cmdline; do echo $(basename $(dirname $proc)) $(cat $proc | tr "\0" " "); done
1 tini -g -- /startup/docker-entrypoint.sh neo4j
1723 bash
7 /opt/java/openjdk/bin/java -cp /var/lib/neo4j/plugins/*:/config/*:/var/lib/neo4j/lib/* -XX:+UseG1GC -XX:-OmitStackTraceInFastThrow -XX:+AlwaysPreTouch -XX:+UnlockExperimentalVMOptions -XX:+TrustFinalNonStaticFields -XX:+DisableExplicitGC -XX:MaxInlineLevel=15 -XX:-UseBiasedLocking -Djdk.nio.maxCachedBufferSize=262144 -Dio.netty.tryReflectionSetAccessible=true -Djdk.tls.ephemeralDHKeySize=2048 -Djdk.tls.rejectClientInitiatedRenegotiation=true -XX:FlightRecorderOptions=stackdepth=256 -XX:+UnlockDiagnosticVMOptions -XX:+DebugNonSafepoints -Dlog4j2.disable.jmx=true -XX:+**ExitOnOutOfMemoryError** -Dfile.encoding=UTF-8 com.neo4j.server.enterprise.EnterpriseEntryPoint --home-dir=/var/lib/neo4j --config-dir=/config --expand-commands

**Review by enys — APPROVED** [review]
thx !

**Comment by thibveni** [comment]
@enys you wanted me to apply it on neo4j v5 ? or not but i forgot why

**Comment by enys** [comment]
yes please

**Comment by thibveni** [comment]
@enys  do you want me to test it for v5? i tried with cd:preview label but it only create a v4 pod

**Comment by thibveni** [comment]
working fine for v5, 
tested on neo4j dev namespace
i merge

**Pull request merged**